modules/tinymce/CHANGELOG.md

```diff
@@ -9,6 +9,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 - New `text_patterns_lookup` option to provide additional text patterns dynamically #TINY-8778
 
+### Improved
+- The autocompleter now supports a multiple character trigger #TINY-8887
```

Comment:
```suggestion
- The autocompleter now supports a multiple character trigger using the new `trigger` configuration #TINY-8887
```
Maybe? As we have no mention about about the new configuration option. We also need to add a `Deprecated` heading below `Fixed` and say that `ch` is now deprecated, use `trigger` instead.

- [ ] Add deprecated changelog entry